src/highlighters.cc

```diff
@@ -1142,7 +1153,10 @@ struct LineNumbersHighlighter : Highlighter
             const auto atom_face = last_line == current_line ? face_wrapped :
                 ((m_hl_cursor_line and is_cursor_line) ? face_absolute : face);
             line.insert(line.begin(), {buffer, atom_face});
-            line.insert(line.begin() + 1, {m_separator, face});
+
+            const auto separator = is_cursor_line && m_separator_cursor ? *m_separator_cursor
```

Comment:
this should be a const-ref.

Reply:
I am not much familiar with C++. I changed the line to
```cpp
const auto& separator { /* ... */ };
```
Is this what you meant?